src/server.c

```diff
@@ -53,11 +53,25 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
     const int max_response_size = 262144;
     char response[max_response_size];
 
-    // Build HTTP response and store it in response
+    int header_length = strlen(header);
+    int content_length = strlen(body);
+    int response_length = header_length + content_length;
 
-    ///////////////////
-    // IMPLEMENT ME! //
-    ///////////////////
+    // time_t rawtime;
+    // struct tm *info;
+    // char buffer[80];
+    // time(&rawtime);
+    // info = localtime(&rawtime);
+
+    // Build HTTP response and store it in response
+    sprintf(response, "%s\n"
+            // "Local-Time: %s\n"
+            "Content-Type: %s\n"
+            "Content-Length: %d\n"
+            "Connection: close\n"
+            "\n"
+            "%s",
+            content_type, /*asctime(info),*/ header, content_length, body);
 
     // Send it all!
     int rv = send(fd, response, response_length, 0);
```

Comment:
Does your "response_length" variable account for the 
`"Content-Type: %s\n"
 "Content-Length: %d\n"
  "Connection: close\n`

lines of your `sprintf()`?